vizro-core/src/vizro/models/_components/title.py

```diff
@@ -0,0 +1,50 @@
+from typing import Annotated, Literal
+
+import dash_bootstrap_components as dbc
+from dash import dcc, html
+from pydantic import AfterValidator, Field
+
+from vizro.models import VizroBaseModel
+from vizro.models._models_utils import _log_call
+
+
+def validate_icon(icon) -> str:
+    return icon.strip().lower().replace(" ", "_")
+
+
+class Title(VizroBaseModel):
+    """Creates a title component.
+
+    Args:
+        type (Literal["title"]): Defaults to `"title"`.
+        text (str): Dashboard title to appear on every page on top left-side. Defaults to `""`.
+        icon (str): Icon name from [Google Material icons library](https://fonts.google.com/icons).
+            Defaults to `"info"`.
+        tooltip (str): Markdown string to create text that appears on icon hover.
+
+    """
+
+    type: Literal["title"] = "title"
+    text: str = Field(description="Dashboard title to appear on every page on top left-side.")
+    icon: Annotated[
+        str,
+        AfterValidator(validate_icon),
+        Field(default="info", description="Icon name from Google Material icons library."),
+    ]
+    tooltip: str = Field(description="Markdown string to create text that appears on icon hover.")
+
+    @_log_call
+    def build(self):
+        return html.Div(
+            id="dashboard-title",
+            children=[
+                html.H2(id="dashboard-title-text", children=self.text),
+                html.Span(self.icon, className="material-symbols-outlined", id=f"{self.id}-icon"),
+                dbc.Tooltip(
+                    id=f"{self.id}-tooltip",
```

Comment:
Two more arguments we should consider here:
* let's set `autohide=False` so it's possible to select text/click links/etc. in the tooltip unless this causes other problems
* we have `fade=True` by default but I don't actually see any animated fade in/out, do you? 🤔 Doesn't really matter but our CSS is somehow accidentally disabling the animation then I'd be curious to see what it looks like. Not important though.